README.md

```diff
@@ -212,6 +212,12 @@ we suggest you check out the following:
 + Interactive Workshop: https://github.com/jlord/git-it-electron
 + [@NataliaLKB](https://github.com/NataliaLKB)'s Tutorial: https://github.com/NataliaLKB/learn-git-basics
 
+### Design
```

Comment:
@harrygfox looks good. 😍 
I'd _suggest_ splitting out the sentences into new lines for ease of reading. see: http://rhodesmill.org/brandon/2012/one-sentence-per-line
@newswim thanks for reviewing. ❤️